github-profiles/KevinC-7394/main.css

```diff
@@ -0,0 +1,25 @@
+body {
```

Comment:
You can include here the div styles (aling:center) to prevent all the repetitions in the html file

Reply:
Your suggestion is helpful, and I'll take it into consideration as I make further CSS updates.